webview/src/shared/components/icons/ChevronRight.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react'
-import { SVGProps } from 'react'
-const SvgChevronRight = (props: SVGProps<SVGSVGElement>) => (
-  <svg
```

Comment:
When I ran `svgr` for the first time, I got three changes to the branch:

* `icons/index.ts` being sorted into a different order
* `ChevronRight.tsx` got a new `<svg` line which I now see was missing. 
* `Check` and `ArrowDown` losing new lines

Are these changes normal/expected or could their be an issue with how `generage.mjs` works?

Reply:
It's probably because I merge "icons-cleanup" back into this branch and fixed conflicts manually. I'll clean this up.

Reply:
But yes, it is expected and wanted. It signals some manual changes were done and are being overwritten. I thought the `postinstall` script would run on the CI, but it might not since the `yarn.lock` didn't change between builds. I'll look into a way to make sure it runs.

Reply:
> `icons/index.ts` being sorted into a different order

This should be normalized though, I'll fix it.

Reply:
Added the script as a step to our CI and fixed the order of the `index.ts` file.